refinery/ui/source/js/commons/services/site-profile-service.spec.js

```diff
@@ -41,11 +41,11 @@ describe('Common.service.siteProfileV2Service: unit tests', function () {
         .expectGET(
           refinerySettings.appRoot +
           refinerySettings.refineryApiV2 +
-          '/site_profiles/'
+          '/site_profiles/?current_site=true'
       ).respond(200, siteProfile);
 
       var results;
-      var promise = service.query().$promise
+      var promise = service.query({ current_site: 'true' }).$promise
```

Comment:
For params being sent, it appears we typically sent it to default to django's 'True'. Based on the style guide, https://github.com/refinery-platform/refinery-platform/wiki/coding-style-guide#rest-api   since we're doing snake_case for params I'd extend it to booleans also.

Reply:
Will require API updates.